app/models/admin_dashboard_data.rb

```diff
@@ -35,7 +35,7 @@ def self.from_h(h)
   GLOBAL_REPORTS ||= []
 
   PROBLEM_MESSAGE_PREFIX = "admin-problem:"
-  SCHEDULED_PROBLEM_STORAGE_KEY = "admin-found-scheduled-problems"
+  SCHEDULED_PROBLEM_STORAGE_KEY = "admin-found-scheduled-problems-list"
```

Comment:
This is my solution for switchover. Use a new key and let the existing key expire. This avoids issues with type-specific operations in Redis.